subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -68,12 +68,14 @@ static void handle_wifi_scan_result(struct net_mgmt_event_callback *cb)
 
 	if (scan_result == 1U) {
 		print(context.sh, SHELL_NORMAL,
-		      "\n%-4s | %-32s %-5s | %-4s | %-4s | %-5s    | %s\n",
-		      "Num", "SSID", "(len)", "Chan", "RSSI", "Sec", "MAC");
+		      "\n%-4s | %-32s %-5s | %-13s | %-4s | %-15s | %s\n",
```

Comment:
The changes in wifi_shell.c will come with an upmerge, let's not introduce them here. And for the future - you should rather cherry-pick upstream commits and tag them with `[nrf fromtree]` prefix instead of creating your own commits.

Reply:
Sure, done. I have dropped these changes as they are being taken care in upmerge.